plugins/formal-verification/formver.plugin/src/org/jetbrains/kotlin/formver/ViperPoweredDeclarationChecker.kt

```diff
@@ -16,19 +16,22 @@ import org.jetbrains.kotlin.fir.declarations.FirContractDescriptionOwner
 import org.jetbrains.kotlin.fir.declarations.FirSimpleFunction
 import org.jetbrains.kotlin.fir.declarations.hasAnnotation
 import org.jetbrains.kotlin.formver.conversion.ProgramConverter
-import org.jetbrains.kotlin.formver.viper.ConsistencyError
-import org.jetbrains.kotlin.formver.viper.VerificationError
 import org.jetbrains.kotlin.formver.viper.Verifier
-import org.jetbrains.kotlin.formver.viper.VerifierError
 import org.jetbrains.kotlin.formver.viper.ast.Program
+import org.jetbrains.kotlin.formver.viper.ast.isEmpty
+import org.jetbrains.kotlin.formver.viper.ast.unwrap
+import org.jetbrains.kotlin.formver.viper.errors.ConsistencyError
+import org.jetbrains.kotlin.formver.viper.errors.VerificationError
+import org.jetbrains.kotlin.formver.viper.errors.VerifierError
 import org.jetbrains.kotlin.name.ClassId
 import org.jetbrains.kotlin.name.FqName
 import org.jetbrains.kotlin.name.Name
 
-private val VerifierError.error: KtDiagnosticFactory1<String>
+private val VerifierError.toPluginError: KtDiagnosticFactory1<String>
```

Comment:
nit: I think calling a property `toX` is a bit strange, that sounds more like a function.  Either a property `asPluginError` or a function `toPluginError` would make more sense to me.